components/style_traits/cursor.rs

```diff
@@ -94,12 +93,13 @@ define_cursor! {
         "all-scroll" => AllScroll = 32,
         "zoom-in" => ZoomIn = 33,
         "zoom-out" => ZoomOut = 34,
+        "auto" => Auto = 35,
```

Comment:
Are these values arbitrary or do they represent something else internally? Is it okay if we just move them by one like this?

Reply:
I don't see them used anywhere, so we could even remove them and simplify the macro, I'd think.

Reply:
Maybe I found something. These values are used within the `impl Cursor` block to define the following function:
https://github.com/servo/servo/blob/671b69c0b77f9a4bd0c098cb2a2f73c95dacb954/components/style_traits/cursor.rs#L38-L45 Then, there's only one place where this function is used: https://github.com/servo/servo/blob/671b69c0b77f9a4bd0c098cb2a2f73c95dacb954/components/compositing/compositor.rs#L772-L777 But because the argument is casted to `_`, it will never match anything and always return an error, which is then converted to `Option` by `.ok()`. Therefore, the `if let` will always evaluate to `None` regardless of the argument. Looks like a bug if I'm not missing anything :).